### DISCUSSION TITLE
Availability of Bottlerocket in China (Beijing) (cn-north-1) or China (Ningxia) (cn-northwest-1) region.

### DISCUSSION THREAD

**Discussion by rajboruah** [bottlerocket-os/bottlerocket#1919] [General]
Hi,

As per https://docs.aws.amazon.com/eks/latest/userguide/launch-node-bottlerocket.html - "You can't deploy to the following regions: China (Beijing) (cn-north-1) or China (Ningxia) (cn-northwest-1)." May I get clarity on this please ? We want to use Bottlerocket as part EKS's _managed node group_ but as I mentioned above, the images does not seem to be available in China (Beijing) (cn-north-1) or China (Ningxia) (cn-northwest-1). Can you please confirm on this please ?

**Comment by zmrow** [comment]
We are not currently available in China - please follow #1255 for updates :)
